### PR TITLE
Add basic template contents

### PR DESCRIPTION
**Pull request by alpgarcia**
First version of the main template documents.

The main goal is having a template and an example at the same time, so please have that in mind while reviewing and look for inconsistencies or aspects that should be remarked.

Thanks!

### CODE REVIEW

File: doc/governance.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Collaboration tools
+
+This project is self-content, so the tools to contribute and participate are GitHub Issues and Pull Request. For completeness, the table below shows an example of common tools by type, with specific application instances. It should be easy to fill in the table for your project:​
+
+| Type | Tools |
+|-     | -     |
+| Source Code Management​ | GitHub.com​ |
+| Issues Tracker​ | JIRA Software by Atlassian (SaaS) <br />GitHub Issues​ |
+| Documentation Platform​ | Confluence Software by Atlassian (SaaS)<br />GitHub Pages<br />Microsoft Teams dedicated channels​ |
+| Forums or Chat Platform​ | Microsoft Teams​ |
```
Comment (dicortazar) [inline doc/governance.md:53]: We could add Slack, Mattermost, Rocketchat as options.

File: doc/governance.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| Metrics  & KPIs Platform​ | GrimoireLab |
+| Projects single entry point​ | Portal in Microsoft Teams​ |
```
Comment (dicortazar) [inline doc/governance.md:55]: We could link to other options as the SAP open source solution for portals and discoverability.

File: doc/governance.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+| Announcements and news​ | Portal in Microsoft Teams <br /> Microsoft Teams channel​ |
```
Comment (dicortazar) [inline doc/governance.md:56]: Again too specific. We should go and detail others.

**Review by dicortazar — CHANGES_REQUESTED** [review]

File: doc/governance.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Given its own nature, it is open to anyone for improvements and contributions, so anyone can propose changes and additions just by following the [contribution guidelines](/CONTRIBUTING.md).
+
+
+## Roles
+
+Find below a common set of roles that could be defined for any project.
+
+| Role  | Members | Objective |
+| -     | -       | -         |
+| Trusted Committer  | Daniel Izquierdo<br />Alberto Pérez<br />Jesús Alonso  | Is the Host Team. Responsible for both building the house and setting the house rules to ensure their guests are comfortable and can work together effectively.  |
```
[the remark this answers is not in the document]
Comment (alpgarcia) [reply doc/governance.md:24]: I did it at first, but thinking it twice, this project has also TCs and should be listed for anyone willing to contribute.

File: doc/governance.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+**It is worth noting that the Champion is added here for reference**. This project does not need it, but it could be useful for others, so we keep it here as a template.
+
+## Forums and committees
+
+This project doesn't have any meeting, nevertheless below there is a list of meetings at the project level and at the methodology level that could generally apply to most of the projects, so you can use this list as starting point.
+
+| Meeting  | Level | Forum Profile | Attendees | Periodicity |
+| -        | -     | -             | -         | -           |
+| Checkpoint​ | Initiative​ | Management​ | Project members | Weekly​ |
+Project Board​ | Project​ | Management / Technical​ | Domain Leader<br />Trusted Committer<br />Champion<br />Technical expert | Ad hoc |​
```
[the remark this answers is not in the document]
Comment (alpgarcia) [reply doc/governance.md:37]: I had sworn I changed this...

File: doc/governance.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+**Project level meetings** have the goal of bring business and technology at the same level, educate stakeholders, align expectations, detect bottlenecks.​
+
+## Collaboration tools
+
+This project is self-content, so the tools to contribute and participate are GitHub Issues and Pull Request. For completeness, the table below shows an example of common tools by type, with specific application instances. It should be easy to fill in the table for your project:​
+
+| Type | Tools |
+|-     | -     |
+| Source Code Management​ | GitHub.com​ |
+| Issues Tracker​ | JIRA Software by Atlassian (SaaS) <br />GitHub Issues​ |
```
[the remark this answers is not in the document]
Comment (alpgarcia) [reply doc/governance.md:51]: Well, these are just some examples not a complete list of options, I just copied them from another project.

File: doc/governance.md (hunk 3 above)
Comment (alpgarcia) [reply doc/governance.md:56]: We can use others, but I don't want to list options here but provide an example of how a project would look like.

**Comment by alpgarcia** [comment]
Thanks for the review @dicortazar!

I've addressed your comments in a new comment and also discussed some of your proposals above.

**Comment by dicortazar** [comment]
This works for me, thanks a lot @alpgarcia !

**Pull request merged**